src/services/infrastructure/storage-aggregator-resolver/storage.aggregator.resolver.service.ts

```
@@ -178,6 +180,51 @@ export class StorageAggregatorResolverService {
 
+    const platform = await this.entityManager.findOne(Platform, {
+      where: {
+        templatesManager: {
+          id: templatesManagerId,
+        },
+      },
+      relations: {
+        storageAggregator: true,
+      },
+    });
+    if (platform && platform.storageAggregator) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)